### DISCUSSION TITLE
游戏玩家求一个非铝合金无扩展的瀚文

### DISCUSSION THREAD

**Discussion by xdeng** [peng-zhihui/HelloWord-Keyboard#33] [General]
看中的是高性能，全新设计，全新固件，硬件可以很强，侧边的扩展就不要了吧，外观可以非常普通，塑料的就行了，铝合金加工的出了名的贵。
1000以内可以接受，开团算我一个。
是的 我csgo一直白银很可能就是缺这么一个NB的键盘。

**Comment by IceSakuraAzure** [comment]
那你可以考虑 之后自己打板做 外壳可以找未来工厂之类的用abs直接3d打印

**Reply by xdeng** [reply]
> 那你可以考虑 之后自己打板做 外壳可以找未来工厂之类的用abs直接3d打印

这不懒么 团购降成本么

**Comment by xdeng** [comment]
还有个很重要的 支持热插拔按键

**Reply by GongXiaoBei77** [reply]
事实上,原视频6分48秒中左右有展示热拔插轴座,而且不止一次出现在视频中.甚至在DOCS中也有明确的热拔插轴座的文档.

**Reply by xdeng** [reply]
> 事实上,原视频6分48秒中左右有展示热拔插轴座,而且不止一次出现在视频中.甚至在DOCS中也有明确的热拔插轴座的文档.

忘了 才想起

**Reply by Starryccc** [reply]
热插拔现在属于是键盘标配了吧

**Comment by gewanbo** [comment]
有没有可能等板子出来以后，发现刚好有一款旧键盘可以装下这块板子

**Reply by xdeng** [reply]
不会这么巧吧

**Comment by Mehver** [comment]
虽然但是，按你这个需求，感觉没必要。

我也打游戏，也自学硬件。看稚晖君的视频我没打算换掉现在的K95，我的关注点反而在全新的算法带来的简洁的电路原理。用这套固件，自己画PCB的难度被大大降低了。这是稚晖君的原创点和精华所在。

当然，你要是说追求的是这个大横槽的板子，也行吧。没太关注客制化，但这种大开槽在传统方案的键盘上好像也有，不过就得上四层板了，但差距在C端也就反映在成本上。

**Reply by xdeng** [reply]
这不想试下新固件 新扫描方式 新去抖算法么。

**Reply by Mehver** [reply]
那可以先做板子。壳子试试能不能组团得了

**Reply by Mehver** [reply]
廉价壳没那么快，但总会有的

**Reply by IceSakuraAzure** [reply]
壳我刚看了下 3d打印的话 树脂便宜点 估计整个下来三四百够了，abs的话 价格有点夸张，pla应该挺便宜，自己有打印机的就能打，不过键盘的尺寸 30多厘米，超过绝大多数消费级3d打印机的尺寸了，估计只能分件打印，开然后用胶粘，不过这样强度就下降了，对成品的影响也不好说。在其他的 就是亚力克堆叠了，这应该是最廉价的壳子方案了。

**Reply by xdeng** [reply]
3D打印不更贵么   工厂批量生成便宜

**Reply by xdeng** [reply]
或者基于现有的公模 改下PCB兼容下

**Reply by IceSakuraAzure** [reply]
> 3D打印不更贵么 工厂批量生成便宜

工厂注塑要模具的 一个模具几十万 销量没有几万个都是亏，而且因为还是客制化，可能有喜欢75的 有喜欢98的 还有107的，一套模板谁知道能不能满足...而3d打印其实成本不高，我们平时折腾的最多的pla塑料 价格也就是50块钱一公斤，而且可以控制填充率，这个键盘打印下来我觉得能不能用一公斤都是回事

**Reply by Mehver** [reply]
淘宝找3D打印的。没试过热熔积的，光固化，便宜树脂，这个差不多五六百。应该也有热熔积的，毕竟大打印机也就那种店买得起

**Reply by IceSakuraAzure** [reply]
> 树脂有个问题就是几个月可能就开始发黄了 而且比较脆，我个人觉得还是热熔积靠谱点
